src/to_wiki.js

```diff
@@ -152,8 +152,13 @@ class WikiTextSerializerState {
       let marks = node.marks.map(m => m.type.name)
 
       let firstNeedle = function(haystack, needles) {
-         return needles.reduce((min, needle) =>
+         const result = needles.reduce((min, needle) =>
           Math.min(min, haystack.indexOf(needle)), haystack.length)
+         if (result === -1) {
+            throw new Exception("Don't use this function on `needles` that " +
```

Comment:
Does this need to be `throw new Error`? I can't find anything in the JS docs about an Exception object.